src/Uno.Toolkit.UITest/Controls/SafeArea/Given_SafeArea.cs

```diff
@@ -18,7 +18,8 @@
 
 namespace Uno.Toolkit.UITest.Controls.SafeArea
 {
-	[TestFixture]
+	[TestFixture]
+	[ActivePlatforms(Platform.iOS, Platform.Browser)]
```

Comment:
wasm doesnt look good atm..:
```suggestion
	[ActivePlatforms(Platform.iOS)]
```